### PR TITLE
Add classes

### PR DESCRIPTION
**Pull request by anagudelogu**
# In this Milestone, we:

- Refactor our script, in order to use classes.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -58,16 +58,16 @@ class Library {
 
   static createBook(book) {
     const bookContainer = div.cloneNode(true);
-    const bookTitle = p.cloneNode(true);
-    const bookAuthor = p.cloneNode(true);
+    const bookText = p.cloneNode(true);
+    // const bookAuthor = p.cloneNode(true);
```
Comment (Kingstalux) [inline script.js:62]: - [ ] Kindly remove all commented code to make your work clean 👍

File: script.js [hunk 2]
```diff
@@ -99,8 +99,8 @@ addButton.addEventListener('click', () => {
 });
 
 bookStorage.addEventListener('click', (e) => {
-  const title = e.target.parentNode.children[0].textContent;
-  const author = e.target.parentNode.children[1].textContent;
+  const text = e.target.parentNode.children[0].textContent;
+  // const author = e.target.parentNode.children[1].textContent;
```
Comment (Kingstalux) [inline script.js:103]: - [ ] Same issue here, kindly remove all commented code 👍

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @anagudelogu and @Olamarx, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6e59597451644c7177776c54653363634b682f67697068792e676966

## Highlights
✅ App has the correct functionality 🔥 
✅ Linters are passing 👍 
✅ Good commit messages 👏 

## Required Changes ♻️
_check the comments below_

- [ ] Kindly update your README file using this [Microverse template](https://github.com/microverseinc/readme-template) and make sure you edit it with your information and information about your project 💯 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
### Hi @anagudelogu, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**